.buildkite/scripts/crosscompile.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Can this script be removed and the call for `make -C ... crosscompile` be changed?

Reply:
will be removed after tests

Reply:
this remark and others too